data/table.txt

```diff
@@ -7671,12 +7707,14 @@
 儭 儭 1 1 0 0 1 0 0 0 0 oydu oydu NA 4615
 儮 儮 1 1 0 0 1 0 0 0 0 omdm omhm NA 4614
 儯 儯 1 0 0 0 1 0 0 0 0 ofqf ofqf NA 0
-儰 儰 1 1 0 0 1 0 0 0 0 otif otbf,otif NA 7383
+儰 儰 1 1 0 0 1 0 0 0 0 otif otbf NA 7383
+儰 儰 1 1 0 0 1 0 0 0 0 NA otif NA 0
 儱 儱 1 1 0 0 1 0 0 0 0 oybp oybp NA 4616
 儲 储 1 1 0 0 1 0 0 0 0 oyra oyra NA 16006
 儳 儳 1 1 0 0 1 0 0 0 0 onri onri NA 15263
 儴 儴 1 1 0 0 1 0 0 0 0 oyrv oyrv NA 4052
-儵 儵 1 1 0 0 1 0 0 0 0 olof olhf,olof,xxolh,xxolo NA 4051
+儵 儵 1 1 0 0 1 0 0 0 0 olof olhf,xxolh NA 4051
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099600):

> "xxolh" returns nothing on MacOS. Probably a non-standard mapping.

---

So for `olhf` we have the following characters in libcangjie:

```
絛 绦 1 1 0 0 1 0 0 0 0 olof olhf NA 18066
鯈 鯈 1 1 0 0 1 0 0 0 0 olof olhf,xolhf NA 15271
儵 儵 1 1 0 0 1 0 0 0 0 olof olhf,xxolh NA 4051
倐 倐 1 0 0 0 1 0 0 0 0 olhf olhf,xxxol NA 0
```

This seems to me like x-disambiguation? (with 0, 1, 2 or 3 "x")

---

@yookoala [said]https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099614):

> "oloh" also do not give "儵" at all on MacOS.

---

I don't understand why you're talking about "oloh" here, that's not a code you seem to be commenting on?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.